src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The function currently only handles specific cases for 3, 6, and 7 days, and defaults to 80 for any other number of days. Ensure this logic aligns with the task requirements. If more specific conditions are needed, consider implementing them.